packages/ui/src/theme/tokens/time.ts

```diff
@@ -9,6 +14,11 @@ export interface Time {
 }
 
 export const time: Time = {
+  animation: {
```

Comment:
What is the difference between 'animation' and 'transition'?

Reply:
+1, seems like we can use the same times for both

Reply:
I like it, so we would just have short, medium, long directly under time? Let's go with that.

Reply:
Yea, exactly 💯